code/linassgn/get_dists_for_datasets.m

```diff
@@ -13,13 +13,11 @@
 if ~exist('datasets') error('Datasets does not exist!'); end;
 
 for i = 1:length(datasets)
-  curr = datasets{i};
-  numTranslates = [];
-  % Get the number of translations for the scaling function of the start level for each dimension.
-  for j = 1:size(curr.sampleSupp,1)
-    numTranslates = [numTranslates diff(translationRange(curr.sampleSupp(j,:), curr.wName, curr.startLevel))+1];
```

Comment:
Does this include wavelets? the curr.sampleSupp should be a 2x2 for 2D. Does curr.startLevel change over the resolutions? If it does then its correct